song_rater/tests.py

```diff
@@ -23,79 +45,109 @@ def test_add_song_form(self):
         self.assertEqual(response.status_code, 200)
 
     def test_add_song_post(self):
-        self.client.post('/add-song', {'artist': 'Strawberry', 'title': 'Shortcake'})
+        test_song = {
+            'artist': 'Strawberry',
+            'title': 'Shortcake'
+        }
+        self.client.post('/add-song', test_song)
```

Comment:
Ik ben niet per se fan van een rewrite als dit.  Iets wat prima duidelijk was in 1 regel , wordt nu 5 kortere regels.  

Maar je  komt op een punt dat het een menings kwestie is.

Reply:
True! I was on the fence about this one. Maybe best to use multi-line declarations when it is used twice or more, or when there goes a lot of data in the dict. Changed it back.